cv32e40p/regress/cv32e40p_xpulp_rand_tests.yaml

```diff
@@ -22,54 +22,41 @@ builds:
     cmd: make comp comp_corev-dv
     cfg: pulp
     dir: cv32e40p/sim/uvmt
-    iss: 1
-    cov: 1
 
   uvmt_cv32e40p_pulp_fpu:
     cmd: make comp comp_corev-dv
     cfg: pulp_fpu
     dir: cv32e40p/sim/uvmt
-    iss: 1
-    cov: 1
 
   uvmt_cv32e40p_pulp_fpu_1cyclat:
     cmd: make comp comp_corev-dv
     cfg: pulp_fpu_1cyclat
     dir: cv32e40p/sim/uvmt
-    iss: 1
-    cov: 1
 
   uvmt_cv32e40p_pulp_fpu_2cyclat:
     cmd: make comp comp_corev-dv
     cfg: pulp_fpu_2cyclat
     dir: cv32e40p/sim/uvmt
-    iss: 1
-    cov: 1
 
   uvmt_cv32e40p_pulp_fpu_zfinx:
     cmd: make comp comp_corev-dv
     cfg: pulp_fpu_zfinx
     dir: cv32e40p/sim/uvmt
-    iss: 1
-    cov: 1
 
   uvmt_cv32e40p_pulp_fpu_zfinx_1cyclat:
     cmd: make comp comp_corev-dv
     cfg: pulp_fpu_zfinx_1cyclat
     dir: cv32e40p/sim/uvmt
-    iss: 1
-    cov: 1
 
   uvmt_cv32e40p_pulp_fpu_zfinx_2cyclat:
     cmd: make comp comp_corev-dv
     cfg: pulp_fpu_zfinx_2cyclat
     dir: cv32e40p/sim/uvmt
-    iss: 1
-    cov: 1
 
 # List of tests
 tests:
   corev_rand_pulp_instr_test:
+    testname: corev_rand_pulp_instr_test
```

Comment:
Just a general comment, it is not needed to explicitely tell the testname if the parent item is already the expected actual testname (for backward compatilibility purpose), but I think it will actually be a good practice to actually put that field to make it clear and consistent with other test items that really **need** the testname field...

Reply:
Yes. It was exactly my intention here as well. I did notice it is not required in this case. But I added for consistency.